Z/zbar/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+name = "zbar"
+version = v"0.23.92"
+
+# Collection of sources required to build imagemagick
+sources = [
+    GitSource("https://github.com/mchehab/zbar.git","aac86d5f08d64ab4c3da78188eb622fa3cb07182"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/zbar
+if [[ "${target}" != "*mingw32*" ]]; then
+    # install `autopoint`
+    apk update && apk add gettext-dev
+fi
+install_license COPYING LICENSE.md
+autoreconf -vfi
+update_configure_scripts
+./configure --with-x=disabled \
+            --enable-pthread=yes \
+            --enable-video=no \
+            --with-jpeg=no \
+            --with-imagemagick=yes \
+            --with-python=no \
+            --with-gtk=no \
+            --with-qt=no \
+            --prefix=${prefix} \
+            --build=${MACHTYPE} \
+            --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(;experimental=true))
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("zbarimg", :zbarimg),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="ImageMagick_jll", uuid="c73af94c-d91f-53ed-93a7-00f77d67a9d7"))
+    Dependency(PackageSpec(name="Libiconv_jll", uuid="94ce4f54-9a6c-5748-9c1c-f9c7231a4531"); platforms=filter(!Sys.isapple, platforms))
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")

```

Reply:
Thanks for taking your time to review the code! I think the build script might be fine now.